### PR TITLE
Merge : 1, Makefile fix, 2, DMFT output, 3, hse bug fix

### PR DESCRIPTION
**Pull request by dyzheng**

### CODE REVIEW

File: source/Makefile.vars [hunk 1]
```diff
@@ -1,42 +1,21 @@
...
-#============DeePKS================
-#LIBTORCH_DIR = /home/fortneu49/soft/libtorch
-#LIBNPY_DIR = /home/fortneu49/soft/libnpy/include
+# LIBTORCH_DIR  = /public/software/libtorch
+# LIBNPY_DIR    = /public/software/libnpy
 
 OBJ_DIR = obj
-OBJ_DIR_serial = obj
-NP      = 14
+OBJ_DIR_serial = obj_serial
```
Comment (caic99) [inline source/Makefile.vars:20]: @dyzheng Any update on this change?

**Review by caic99 — APPROVED** [review]

File: source/module_xc/xc_functional.cpp [hunk 2]
```diff
@@ -120,6 +120,7 @@ void XC_Functional::set_xc_type(const std::string xc_func_in)
     {
         func_id.push_back(XC_HYB_GGA_XC_HSE06);
         func_type = 4;
+        use_libxc = true;
```
Comment (caic99) [inline source/module_xc/xc_functional.cpp:123]: This patch is the same as #833 , FYI.

File: source/Makefile.vars (hunk 1 above)
Comment (dyzheng) [reply source/Makefile.vars:20]: I don't know detail about Makefile updates, but linpz suggested that the update in this PR would benefit for users who are willing to use Makefile for compiling.

File: source/module_xc/xc_functional.cpp (hunk 2 above)
Comment (dyzheng) [reply source/module_xc/xc_functional.cpp:123]: #833 has been closed yet.

**Pull request merged**